### PR TITLE
clean docker compose file

### PR DESCRIPTION
**Pull request by Tianhao-Gu**

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -231,6 +160,7 @@ services:
       - NETWORK_NAME=cdm-jupyterhub-network
       - JUPYTERHUB_USER_IMAGE=cdm-jupyterhub-cdm_jupyterhub:latest
       - JUPYTERHUB_MOUNT_BASE_DIR=/cdr/cdm/jupyter
+      - ENVIRONMENT=dev
```
Comment (MrCreosote) [inline docker-compose.yaml:163]: Is there no user hub instance?

File: docker-compose.yaml [hunk 2]
```diff
@@ -57,10 +54,7 @@ services:
   spark-worker-1:
     # The latest image from cdm-jupyterhub that includes spark standalone mode
     image: ghcr.io/kbase/cdm-jupyterhub:pr-74
-    container_name: spark-worker-1
     platform: linux/amd64
-    depends_on:
-      - spark-master
```
[the remark this answers is not in the document]
Comment (Tianhao-Gu) [reply docker-compose.yaml:63]: It really doesn't matter now. I think at one time we might even remove stand alone spark. It slows down docker compose up/down. I can add it back but I think it's okay to leave it out.

File: docker-compose.yaml (hunk 1 above)
Comment (Tianhao-Gu) [reply docker-compose.yaml:163]: no. Just one hub instance and permissions are controlled via user groups.

Comment (MrCreosote) [reply docker-compose.yaml:163]: What does the ENV variable do in that case?

Comment (Tianhao-Gu) [reply docker-compose.yaml:163]: https://github.com/kbase/cdm-jupyterhub/blob/main/config/jupyterhub_config.py#L64-L66

**Review by MrCreosote — APPROVED** [review]

File: docker-compose.yaml (hunk 1 above)
Comment (MrCreosote) [reply docker-compose.yaml:163]: I'd probably give that env var a more descriptive name in that case, like REMOVE_CONTAINER or something

**Pull request merged**